src/KOKKOS/pair_eam_kokkos.cpp

```diff
@@ -31,9 +31,25 @@
 #include "pair_kokkos.h"
 
 #include <cmath>
-
 using namespace LAMMPS_NS;
 
+#define MAX_CACHE_ROWS 500
+#define MAX_CACHE_COLS 7
+
+/* ---------------------------------------------------------------------- */
+template<class DeviceType, class tag>
+auto policyInstance(int inum){
+  #ifdef KOKKOS_ENABLE_HIP
```

Comment:
These optimizations also don't seem particularly special to one type of GPU :)

Reply:
Our Ampere is dead for the moment, haven't been able to test if it matters for you guys :P

Reply:
If you get a chance @weinbe2 -- give this a test, and we can turn it on for y'all (or just LMP_ENABLE_GPU), our A100 box still isn't alive yet.